### ISSUE TITLE
Search by entire phrase?

### ISSUE THREAD

**Issue by dzg** [christian-fei/Simple-Jekyll-Search#182]
Is there any way to make this search for the entire input string, rather the words?

E.g., if I search for "let it roll", I get results for all posts which contain those 3 words, anywhere, in any order. 
I want to find posts which contain the actual string "let it roll".

**Comment by dzg** [comment]
I was able to accomplish this by changing `src/SearchStrategies/LiteralSearchStrategy.js `
to

```js
function LiteralSearchStrategy() {
   this.matches = function (str, crit) {
      if (!str) return false
      str = str.trim().toLowerCase()
      crit = crit.trim().toLowerCase()

      return str.indexOf(crit) >= 0
   }
}

```

but I would vote for having this be a search option.

Thanks! Awesome plugin!

**Comment by dzg** [comment]
... or instead of a flag, it could be triggered when the search input begins/ends with `"` ... like a Google search...

**Comment by dzg** [comment]
I just realized this was already implemented in 2df6280 but it hasn't been compiled to `/dest` yet
